web_app/api/user.py

```diff
@@ -106,6 +107,38 @@ async def update_user_contract(
         return {"is_contract_deployed": False}
 
 
+@router.post(
+    "/api/subscribe-to-notification",
+    tags=["User Operations"],
+    summary="Subscribe user to notifications",
+    response_model=SubscribeToNotificationResponse,
+    response_description="Returns success status of notification subscription",
+)
+async def subscribe_to_notification(
+    data: SubscribeToNotificationResponse,
+) -> SubscribeToNotificationResponse:
+    """
+    This endpoint subscribes a user to notifications by linking their telegram ID to their wallet.
+
+    ### Parameters:
+    - **telegram_id**: The Telegram id of the user.
+    - **wallet_id**: The wallet id of the user.
+
+    ### Returns:
+    Success status of the subscription.
+    """
+    user = user_db.get_user_by_wallet_id(data.wallet_id)
+    if not user:
+        raise HTTPException(status_code=404, detail="User not found")
+
+    # Simulate subscription logic
```

Comment:
Please, add here # TODO with next command: 
` #TODO add crud method to subscribe user to notification`